packages/flutter/test/widgets/scrollable_selection_test.dart

```diff
@@ -190,6 +190,102 @@ void main() {
     await gesture.up();
   }, skip: kIsWeb); // https://github.com/flutter/flutter/issues/125582.
 
+  testWidgets('mouse can select multiple widgets on triple-click drag', (WidgetTester tester) async {
```

Comment:
How do non-text selectable widgets behave in relation to these word/paragraph gestures?  I'm thinking of something like the icon in the example in the [SelectableRegion docs](https://master-api.flutter.dev/flutter/widgets/SelectableRegion-class.html) about "making a widget selectable".  If you have a many widgets like that, do they each behave like a word and a paragraph?